_playwright-tests/tests/API/Test.spec.ts

```diff
@@ -0,0 +1,80 @@
+import { TestConfig, confToAnnotations, test } from './base_client';
+import { RepositoriesApi, GetRepositoryRequest, ApiRepositoryResponse } from './client';
+import { expect } from '@playwright/test';
+import { cleanupRepositories, poll } from './apiHelpers';
+
+test.describe('Test', () => {
+  // --------------- Approach no. 1 ---------------
+  const testConf: TestConfig = {
+    repoNames: ['test-2', 'reponame'],
+    urls: ['https://fedorapeople.org/groups/katello/fakerepos/zoo/'],
+    // templatesNames: [...]
+  };
```

Comment:
i wonder if we could name this something to make it a bit more obvious why its there.  `cleanupConf`  or something like that?

Reply:
defo, I am not the best with naming stuff, and `cleanupConf`|`CleanupConfig` for the type works 👍🏼😄